dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/Reloader.scala

```diff
@@ -193,6 +214,7 @@ class Reloader(
    * - null - If nothing changed.
    */
   def reload: AnyRef = {
+    // fixme: synchronize on something that is guaranteed to be global in mavens multi classloader environment
```

Comment:
Are you planning to fix this before merging? Just pointing it out in case you have forgot about your own note.